data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,367 @@
+				goto leave
+			label "ride offer"
+			`	"Yes! Some crooked human merchants locked us in a cell and took us here. I've been unable to find anyone who has even heard of Hai space, and I'm afraid to approach the Syndicate. You're the first lead I have!"`
+			`	"I promise I'll pay you well when we return. I can afford <payment>... when I'm back at <destination>."`
+			`	"My name is Duniip Yeertle, by the way. My wife and three children are nearby. They have holoprojectors as well."`
+			label leave
+			`	"Follow me back to my house, and we will make our preparations."`
+			choice
+				`	(Follow him.)`
+			`	You walk down a corridor to a stairway that leads to a brick wall. Duniip walks through the wall; apparently this is another of his holograms. You pass through and see a triad of energetic Hai children eating dinner with their mother.`
+			`	Duniip turns off his hologram and addresses his family, "I found us a pilot! He's going to take us all the way home." This was followed by four cheering family members and one baby's gurgle.`
+			`	The family was already packed to leave. Duniip explains they move regularly so they're not found. It takes them under two minutes to finish preparations and don their holoprojectors.`
+			`	Your walk back to the ship is largely uneventful until some men with strange scanners detect the Yeertle family. One man throws a small grenade with a green light on top. You recognize it as a low-level EMP device. All five holograms turn off, along with all computers within twenty meters. Fortunately, your ship is outside that range.`
+			`	The six of you run to your ship as quickly as possible. You look back and see the men were not following you; they were running to their own ships.`
+				launch
+	on complete
+		payment 350000
+		"hai slave prereq" ++
+		dialog `The Yeertle family is ecstatic to be home after their long ordeal and gladly give you <payment>. They promise to look at the background checks of human pilots before hiring next time.`
+	npc
+		government "Bounty Hunter that Won't Enter Hai Space"
+		personality plunders disables unconstrained
+		fleet "Large Core Pirates" 1
+		fleet "Small Core Pirates" 1
+
+
+mission "Hai Rescue: Prison Role-Players"
+	# You get to rescue the Hai from "First Contact: Hai"
+	landing
+	name "Return the Prison Role-Players"
+	description "Return Hai prison role-players to <destination>. Payment is <payment>."
+	source "Sunracer"
+	destination "Allhome"
+	passengers 5
+	repeat 3
+	to offer
+		has "First Contact: Hai: declined"
+		"Hai Rescue: Prison Role-Players (offer count)" < 3
+		not "event: hai-human treaty signed"
+		not "Hai Rescue: Prison Role-Players: offered"
+		# No combat rating requirement since there are no NPCs.
+		has "leaks response 5: accepted"
+		# Some subset of these are restricted so you can't pick up enough of them to finish the quest at the same time as starting it.
```

Comment:
I quite liked the idea that if you knew about the Hai, you could discover some of the kidnapped Hai missions before the events broke. So I made the two most distant ones able to be done whenever, so that the player might already have a bit of background to hook them in (and because, being the most distant, they're the least likely to get done otherwise, so letting them be available earlier gives more opportunity for the content to be encountered equally.
Before the main story, they're isolated incidents, after the main story, they're suddenly part of the web of events that tie together.
But I only made 2 of them able to be found at that point, because I wanted the player to still have to go and do at least 1 mission at the time it came up in order to meet the 3 required and progress the story.